fdbserver/Ratekeeper.actor.cpp

```diff
@@ -537,7 +547,8 @@ struct RatekeeperData {
 ACTOR Future<Void> trackStorageServerQueueInfo( RatekeeperData* self, StorageServerInterface ssi ) {
 	self->storageQueueInfo.insert( mapPair(ssi.id(), StorageQueueInfo(ssi.id(), ssi.locality) ) );
 	state Map<UID, StorageQueueInfo>::iterator myQueueInfo = self->storageQueueInfo.find(ssi.id());
-	TraceEvent("RkTracking", ssi.id());
+	TraceEvent("RkTracking", self->id)
+		.detail("SSID", ssi.id());
```

Comment:
I think I would prefer these names being a little more explicit, such as:

```suggestion
		.detail("StorageServer", ssi.id());
```

Reply:
I changed these (and the `TTID` to `TransactionLog`)